Removed remaining references to deprecated tutorial (#1439)
spatialos/UnrealGDK: SpatialGDK/Documentation/index.md

#### **> Next:** [Get started]({{urlRoot}}/content/get-started/introduction.md)

</br>------</br>

> _2019-10-16 Updated with editorial review: removed mention of the deprecated tutorial -  multiple deployments for session-based games_</br>

Reviewer: ```suggestion
_2019-10-18 Page updated with editorial review: removed mention of the deprecated tutorial - multiple deployments for session-based games_</br>
```

> _2019-08-09 Updated with editorial review_</br>

Reviewer: ```suggestion
_2019-08-09 Page updated with editorial review_</br>
```

> _2019-08-08 Updated with editorial review: renamed "multiserver shooter tutorial" to "multiserver zoning tutorial"_ </br>

Reviewer: ```suggestion
_2019-08-08 Page updated with editorial review: renamed "multiserver shooter tutorial" to "multiserver zoning tutorial"_ </br>
```

> _2019-07-31 Updated with limited editorial review_

Reviewer: ```suggestion
_2019-07-31 Page updated with limited editorial review_
```
